src/services/FanartApi.ts

```diff
@@ -0,0 +1,37 @@
+import axios, {AxiosInstance} from 'axios';
+
+export default class FanartApi {
```

Comment:
Should make this implement an interface to make testing via mocking easier, etc.

Reply:
Same with `TraktApi`